spec/takeaway_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu: menu, order: order) }
+  let(:menu)         { double(:menu, show: show_menu) }
+  let(:order)        { double(:order, total: 15.50) }
+  let(:show_menu)    { 'All the dishes' }
+  let(:dishes)       { {starter: 2, main: 4} }
+
+  context ' #menu' do
+    it 'so a customer can see a list of dishes, it has a menu including those dishes' do
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)